EpicLoot/EpicLoot.cs

```diff
@@ -915,9 +970,63 @@ void Process()
 	            watcher.IncludeSubdirectories = true;
 	            watcher.SynchronizingObject = ThreadingHelper.SynchronizingObject;
 	            watcher.EnableRaisingEvents = true;
+
+                //Patch JSON Watcher
+                for (var i = 0; i < FilePatching.PatchesPerFile.Where(y => y.Key.Equals(filename)).ToList().Count; i++)
+                {
+                    var configFile = FilePatching.PatchesPerFile.Where(y => y.Key.Equals(filename)).ToList()[i];
+                    var lists = configFile.Value.Select(p => p.SourceFile).Distinct().ToList();
+
+                    for (var index = 0; index < lists.Count; index++)
+                    {
+                        var patchfile = lists[index];
+                        AddPatchFileWatcher(filename,patchfile);
+                        
+                    }
+                }
+            }
+        }
+
+        private static void AddPatchFileWatcher(string fileName, string patchFile)
+        {
+            var fullPatchFilename = string.Concat(FilePatching.PatchesDirPath, patchFile);
```

Comment:
Use Path.Combine

Reply:
Okay, so unless I do additional string manipulation, the SoureName that is being used across the board in your filepatching is prefixing the name with a "\".  Path.Combine isn't handling that well throwing errors.   I have to manipulate it and remove the leading slash, to get it to work.. or I can just used string. Concat, which is working as expected.

Line 206 of the FilePatching, is listed as such:
`                patch.SourceFile = file.FullName.Replace(PatchesDirPath, "");`

This is what's causing the leading slash in SourceFile.. I didn't want to change this in case it was being referenced elsewhere.

Reply:
Hmm no that's actually a bug, I didn't want to keep the leading slash, but I did want to keep the relative filepath to the dll

Reply:
Ahh ok... So I have fixed the bug and removed the string.Concat with Path.Combine's.